pkg/dotc1z/sql_helpers.go

```diff
@@ -335,40 +336,104 @@ func executeChunkedInsert(
 		}
 		chunkedRows := rows[start:end]
 
-		// Create the base insert dataset
+		err := executeChunkWithRetry(ctx, c, tableName, chunkedRows, buildQueryFn)
```

Comment:
Previously this code inserted all the chunks in one big transaction. This change inserts each chunk as a transaction. I'm pretty sure we don't want that.